classes/PrestaShopAutoload.php

```diff
@@ -154,7 +154,10 @@ public function generateIndex()
 		if ($filename_tmp !== false && file_put_contents($filename_tmp, $content) !== false)
 		{
 			if (!@rename($filename_tmp, $filename))
+			{
```

Comment:
Hi @jmaumene,

Can you please make it PSR2 compliant by pulling this brackets next to the end of prior "if" statement ?

Thanks :)